share/spice/twitch/featured/twitch_featured.js

```diff
@@ -0,0 +1,44 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_twitch_featured = function(api_result){
+
+        if (!api_result || !api_result.featured || api_result.featured.length === 0 ||api_result.error) {
+            return Spice.failed('twitch_featured');
+        }
+        
+        Spice.add({
+            minItemsForModeSwitch: 3,
+            id: "twitch_featured",
+            name: "Twitch Featured Streams",
+            data: api_result.featured,
+            meta: {
+                sourceName: "twitch.tv",
+                sourceUrl: "http://www.twitch.tv/",
+                itemType: "Twitch Featured"
+            },
+            templates: {
+                group: 'base',
+                detail: false,
+                item_detail: false,
+                item: 'videos_item',
+                moreAt: true,
+            },
+            normalize: function(featuredStream) {
```

Comment:
`itemType` should probably be "Featured Streams" so that it would read "Showing 25 Featured Streams". 

What I was referring to here was the argument `featuredStream` should just be called `item`--just for consistency with the other Spice IAs.